mummer/4.0.0/Dockerfile

```diff
@@ -0,0 +1,75 @@
+# FROM defines the base docker image. This command has to come first in the file
+# The 'as' keyword lets you name the following stage. We use `app` for the production image
+FROM ubuntu:xenial as app
+
+# ARG sets environment variables during the build stage
+ARG MUMMER_VER="4.0.0"
+
+# LABEL instructions tag the image with metadata that might be important to the user
+LABEL base.image="ubuntu:xenial"
+LABEL dockerfile.version="1"
+LABEL software="mummer"
+LABEL software.version=$MUMMER_VER
+LABEL description="MUMmer is a versatile alignment tool for DNA and protein sequences."
+LABEL website="https://github.com/StaPH-B/docker-builds"
+LABEL license="https://github.com/StaPH-B/docker-builds/blob/master/LICENSE"
+LABEL maintainer="John Arnn"
+LABEL maintainer.email="jarnn@utah.gov"
+
+# RUN executes code during the build
+# Install dependencies via apt-get or yum if using a centos or fedora base
+RUN apt-get update && apt-get install -y \
+ wget \
+ git \
+ libncurses5-dev \
+ libbz2-dev \
+ liblzma-dev \
+ libcurl4-gnutls-dev \
+ zlib1g-dev \
+ libssl-dev \
+ gcc \
+ make \
+ perl \
+ bzip2 \
+ gnuplot \
+ ca-certificates \
+ gawk \
+ curl \
+ sed \
+ sudo \
+ build-essential && \
+ apt-get autoclean && rm -rf /var/lib/apt/lists/*
+
+ # install mummer
+RUN wget https://github.com/mummer4/mummer/releases/download/v${MUMMER_VER}rc1/mummer-${MUMMER_VER}rc1.tar.gz && \
+  tar -xvf mummer-${MUMMER_VER}rc1.tar.gz && \
+  rm mummer-${MUMMER_VER}rc1.tar.gz && \
+  cd mummer-${MUMMER_VER}rc1 && \
+  ./configure && \
+  make && \
+  make install && \
+  sudo ldconfig && \
+  cd .. && \
+  mkdir /data
+
+ENV PATH="/usr/local/bin:$PATH" \
```

Comment:
line 55 may be unnecessary, I'm 99% sure `/usr/local/bin` is in PATH be default. Doesn't hurt to keep it though, just sets the priority for that directories over all others in PATH